samples/benchmarks/coremark/sample.yaml

```diff
@@ -81,15 +81,3 @@ tests:
       - nrf54h20dk/nrf54h20/cpuapp
     tags: ci_build sysbuild ci_samples_benchmarks
     extra_args: EXTRA_CONF_FILE="prj_multiple_threads.conf"
```

Comment:
Did we tested that logging works fine also for other configurations: multithread, static, heap?

static and heap shouldn't be affected but multithread can work strange.

Reply:
Could you clarify -  we do not support those configurations right now or does this change break them?

Reply:
No, they were broken since the very beginning